nifty-renderer-libgdx/src/main/java/de/lessvoid/nifty/gdx/render/GdxGL.java

```diff
@@ -1,10 +1,11 @@
 package de.lessvoid.nifty.gdx.render;
 
 import com.badlogic.gdx.Gdx;
-import com.badlogic.gdx.graphics.GL10;
-import com.badlogic.gdx.graphics.GL11;
 
+import com.badlogic.gdx.utils.BufferUtils;
 import de.lessvoid.nifty.render.batch.spi.GL;
+import org.lwjgl.opengl.GL11;
```

Comment:
The only real way to handle this is to change the shaders when rendering components with specific alpha testing. Everything else is not supported anymore in OpenGL 2.0

Basically you need a shader that discards the fragment once a specific threshold of the alpha value is passed.

Something like:
```
if (color.a < alphaMin) {
    discard;
}
```